joomla/joomla-cms
# Custom field description is not displayed in the article or category editor form

Pull Request for Issue # .

### Summary of Changes

The description text of the custom field should be displayed in the article editing window.

![printscreen09-12-22 at 09 40 AM](https://user-images.githubusercontent.com/16303176/189603983-2c3a9893-0248-41a8-95f7-d66c8ff4acd4.JPG)

before without description in Article/Category Editor:
![printscreen09-12-22 at 09 45 AM 001](https://user-images.githubusercontent.com/16303176/189603514-de055624-6d63-413f-be71-f094165b0d94.JPG)

after:
![printscreen09-12-22 at 09 49 AM](https://user-images.githubusercontent.com/16303176/189603530-c056b040-872f-4c2b-9823-541b7d363340.JPG)

### Testing Instructions

Go to "Content - Fields". Create a custom field, e.g. textarea. Insert a text in the field "Description" and save and close.

![printscreen09-12-22 at 09 40 AM](https://user-images.githubusercontent.com/16303176/189602859-b63b1c93-bbe9-4b38-8219-7b8d69f02c75.JPG)

Go to Content - Articles. Create or edit an article. In the "Fields" tab, it should display the description text for the custom field.

![printscreen09-12-22 at 09 45 AM](https://user-images.githubusercontent.com/16303176/189603003-e34bee40-a2c1-49f6-978c-e95f2892b536.JPG)

The same should be done for the custom fields of the categories.


### Actual result BEFORE applying this Pull Request
The field description of the custom field is not displayed in the Article or Category form.

![printscreen09-12-22 at 09 45 AM 001](https://user-images.githubusercontent.com/16303176/189603084-e17c3e5e-20ac-4158-a370-c8f6f542bed5.JPG)


### Expected result AFTER applying this Pull Request

![printscreen09-12-22 at 09 49 AM](https://user-images.githubusercontent.com/16303176/189603119-70b4c0a2-f47e-4d74-9734-e49562164e31.JPG)


### Documentation Changes Required
